prow/config/presets.yaml

```diff
@@ -74,3 +74,32 @@ presets:
   - name: creds-gcs
     secret:
       secretName: gcs-credentials
+
+- labels:
+    preset-aws: "true"
+  env:
+  - name: AWS_DEFAULT_PROFILE
+    valueFrom:
+      secretKeyRef:
+        name: aws
+        key: AWS_DEFAULT_PROFILE
+  - name: AWS_ACCESS_KEY_ID
+    valueFrom:
+      secretKeyRef:
+        name: aws
+        key: AWS_ACCESS_KEY_ID
+  - name: AWS_SECRET_ACCESS_KEY
+    valueFrom:
+      secretKeyRef:
+        name: aws
+        key: AWS_SECRET_ACCESS_KEY
+  - name: OCP_PULL_SECRET
+    valueFrom:
+      secretKeyRef:
+        name: aws
```

Comment:
The OCP_PULL_SECRET is named `aws`?

Reply:
Same with Quay

Reply:
the secret is

Reply:
I mean, it's one single secret named aws with multiple entries in its data field.

Reply:
Secret looks like this:
```yaml
apiVersion: v1
kind: Secret
metadata:
  name: aws
type: Opaque
stringData:
  OCP_PULL_SECRET: ''
  QUAY_PULL_SECRET: ''
  AWS_ACCESS_KEY_ID: ''
  AWS_SECRET_ACCESS_KEY: ''
  AWS_DEFAULT_PROFILE: ''
```